inference-engine/src/low_precision_transformations/include/low_precision/transformer.hpp

```diff
@@ -44,6 +44,21 @@ class TRANSFORMATIONS_API LowPrecisionTransformations {
     LowPrecisionTransformations& removeTransformations(const std::string& operationType);
     LowPrecisionTransformations& removeCleanupTransformations(const std::string& operationType);
 
+    template <class Transformation, class Operation>
+    LowPrecisionTransformations& removeStandaloneCleanup() {
```

Comment:
Maybe `removeStandaloneCleanupTransformations` would be more consistent with other methods?